plugins/window-state/src/lib.rs

```diff
@@ -309,6 +317,7 @@ pub struct Builder {
     denylist: HashSet<String>,
     skip_initial_state: HashSet<String>,
     state_flags: StateFlags,
+    map_label: Option<Box<dyn (Fn(&str) -> &str) + Send + Sync>>,
```

Comment:
extra parentheses is not needed 
```suggestion
    map_label: Option<Box<dyn Fn(&str) -> &str + Send + Sync>>,
```